LoanCalc.java

```diff
@@ -38,17 +50,36 @@ private static double endBalance(double loan, double rate, int n, double payment
 	// the number of periods (n), and epsilon, the approximation's accuracy
 	// Side effect: modifies the class variable iterationCounter.
     public static double bruteForceSolver(double loan, double rate, int n, double epsilon) {
-		// Replace the following statement with your code
-		return 0;
-    }
+		double g = loan/n;
```

Comment:
Missing `iterationCounter = 0`
if you would have ran brute function twice in one run. it will be incorrect